src/sentry/static/sentry/app/plugins/components/settings.jsx

```diff
@@ -94,6 +95,13 @@ class PluginSettings extends PluginComponentBase {
     }
```

Comment:
I probably would have aliased `this.state` at this point, given how much it is used repeatedly in this function. One benefit is that ESLint will catch misspelling the variable whereas misspelling the property (e.g. `this.tsate`) wouldn't.

(Not a blocker, just a comment.)